src/api/comment/content-types/comment/schema.json

```diff
@@ -29,6 +29,12 @@
       "relation": "manyToOne",
       "target": "plugin::users-permissions.user",
       "inversedBy": "comments"
+    },
```

Comment:
I think something went wrong during the rebase, because authorId is removed from the main branch https://github.com/dev-launchers/strapiv4/blob/main/src/api/comment/content-types/comment/schema.json

Reply:
Yeah, I don't know very well what is wrong.

So to rebase main, is this the rigth command ?

```
➜  strapiv4-api git:(tiago/124-fix-remove-point-collection) git rebase main

Current branch tiago/124-fix-remove-point-collection is up to date.
```

Reply:
The command looks right. One possibility is that your local main branch was not updated. You can also fix this by removing the authorId field manually.